var/spack/repos/builtin/packages/py-trimesh/package.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyTrimesh(PythonPackage):
+    """Import, export, process, analyze and view triangular meshes"""
+
+    homepage = "https://github.com/mikedh/trimesh"
+    pypi = "trimesh/trimesh-2.38.10.tar.gz"
+
+    version("3.17.1", sha256="025bb2fa3a2e87bdd6873f11db45a7ca19216f2f8b6aed29140fca57e32c298e",)
+    version("2.38.10", sha256="866e73ea35641ff2af73867c891d7f9b90c75ccb8a3c1e8e06e16ff9af1f8c64",)
+
+    variant(
+        "easy",
+        default=False,
+        description="Install soft dependencies and unlock extra functionality")
+
+    depends_on("py-setuptools@40.8:", type="build")
+
+    depends_on("py-numpy", type=("build", "run"))
+    depends_on("py-networkx", type=("build", "run"), when="+easy")
+    depends_on("py-shapely", type=("build", "run"), when="+easy")
+    depends_on("py-scipy", type=("build", "run"), when="+easy")
+    depends_on("py-svgpath", type=("build", "run"), when="+easy")
+    depends_on("py-lxml", type=("build", "run"), when="+easy")
```

Comment:
Missing a lot of deps required for easy:
```
requirements_easy = set([                                                                
    'scipy',     # provide convex hulls, fast graph ops, etc                             
    'networkx',  # provide slow graph ops with a nice API                                
    'lxml',      # handle XML better and faster than built- in XML                       
    'pyglet<2',  # render preview windows nicely : note pyglet 2.0 is basically a re-write
    'shapely',   # handle 2D polygons robustly                                           
    'rtree',     # create N-dimension trees for broad-phase queries                      
    'svg.path',  # handle SVG format path strings                                        
    'sympy',     # do analytical math                                                    
    'msgpack',   # serialize into msgpack                                                
    'pillow',    # load images                                                           
    'requests',  # do network requests                                                   
    'xxhash',    # hash ndarrays faster than built-in MD5/CRC                            
    'setuptools',  # do setuptools stuff                                                 
    'jsonschema',  # validate JSON schemas like GLTF                                     
    'pycollada',   # parse collada/dae/zae files                                         
    'chardet',     # figure out if someone used UTF-16                                   
    'mapbox-earcut',  # fast 2D triangulations of polygons                               
    'colorlog'])   # log in pretty colors              
```

Reply:
That should be okay now.